### PR TITLE
Release/v2.3.4 into master

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/Myddleware/RegleBundle/DataFixtures/ORM/LoadSolutionData.php [hunk 1]
```diff
@@ -37,10 +37,8 @@ class LoadSolutionData implements FixtureInterface
 									array('name' => 'salesforce',		'active' => 1,'source' => 1,'target' => 1),
 									array('name' => 'prestashop',		'active' => 1,'source' => 1,'target' => 1),
 									array('name' => 'dolist',			'active' => 1,'source' => 1,'target' => 1),
-									array('name' => 'eventbrite',		'active' => 0,'source' => 1,'target' => 0),
```
Comment (ghost) [inline src/Myddleware/RegleBundle/DataFixtures/ORM/LoadSolutionData.php:40]: @Myddleware 

Are you sure to delete eventbrite and bittle solution ?

Comment (Myddleware) [reply src/Myddleware/RegleBundle/DataFixtures/ORM/LoadSolutionData.php:40]: Yes Dolyveen, we don't maintain these connectors anymore and nobody use it.
Thanks for your checking.

**Review by Myddleware — DISMISSED** [review]

**Comment by Myddleware** [comment]
We will do non regression test now before the merge into Master

**Comment by ghost** [comment]
Ok, 

After validation, i will merged into master and put the v2.3.4 tag.

**Comment by ghost** [comment]
@Myddleware , you must approving this pull request and after i will could merge it

**Review by Myddleware — APPROVED** [review]
I approve this merge request.

**Pull request merged**